conda_pack/tests/test_core.py

```diff
@@ -131,6 +127,7 @@ def test_file():
     repr(f)
 
 
+@pytest.mark.skipif(on_win, reason='Different filenames and paths on Windows')
```

Comment:
Would be good to modify this test to still work on windows, rather than skipping it. It looks like you just copied and modified it below, I'd prefer to keep the tests together and add conditionals for the differing file paths.

Reply:
doing so now.